commands/utils/set_args.js

```diff
@@ -353,6 +353,14 @@ function sync_args_from_cmd(args) {
       lt_config["run_settings"]["network"] = false;
     }
 
+    if ("network_http2" in args) {
```

Comment:
please check this block

Reply:
in this ternary operation. 

 lt_config["run_settings"]["network_http2"] = true
        ? args["network_http2"] == "true"
        : false;
        
value of this   lt_config["run_settings"]["network_http2"]   will be equal to 

value of args["network_http2"] == "true"

so it is correct.

Reply:
@HRanjan-11 but this will always be the value of args["network_http2"] == "true"
so there is not point of having ternary operator. 
you can simply have 

`lt_config["run_settings"]["network_http2"] = args["network_http2"] == "true"`

Reply:
done